packages/havoc-c2/PKGBUILD

```diff
@@ -0,0 +1,84 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+_pkgname=Havoc
+pkgname=havoc-c2
+pkgver=856.fb67b76
+pkgrel=1
+pkgdesc='Modern and malleable post-exploitation command and control framework.'
+arch=('x86_64' 'i686')
+groups=('blackarch' 'blackarch-automation' 'blackarch-backdoor' 'blackarch-exploitation' 'blackarch-networking')
+url='https://havocframework.com/'
+license=('GPL3')
+depends=('openssl' 'qt5-base' 'qt5-websockets' 'python3' 'fontconfig' 'gtest'
+		 'spdlog' 'boost' 'boost-libs' 'ncurses' 'gdbm' 'readline' 'libffi'
+		 'sqlite' 'bzip2' 'nasm' 'mingw-w64-gcc' 'go' 'mesa' 'glu')
```

Comment:
Isn't `go` a build dep?

Reply:
AUR package seems to use `go` as runtime dependency https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=havoc-c2-git#n13

Btw I just directly asked to the developer. He'll back soon with an answer.